### PR TITLE
scheduler proj

### PR DESCRIPTION
**Pull request by rantifamilusi**
add job, trigger and scheduler with cron expression with some unit testing

### CODE REVIEW

File: app/JobManager.Test/JobManager.Test.csproj [hunk 1]
```diff
@@ -1,15 +1,20 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (alexjoybc) [inline app/JobManager.Test/JobManager.Test.csproj]: Ahaha

File: app/JobManager/Program.cs [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+using System.Threading.Tasks;
 
 namespace JobManager
 {
     class Program
     {
-        static void Main(string[] args)
+        static async Task  Main(string[] args)
         {
+            IScheduler sched = await new PersonToSearchScheduleEngine().CreateScheduler();
```
Comment (alexjoybc) [inline app/JobManager/Program.cs]: Can we use full name for variables?

File: app/JobManager.Test/JobManager.Test.csproj (hunk 1 above)
Comment (rantifamilusi) [reply app/JobManager.Test/JobManager.Test.csproj]: Not sure what to change here... i changed it from 3.0 to 2.2.. Don't know why VS 2019 is creating automatically for 3.0. Maybe MSFT wants us to move to 3.0

File: app/JobManager/Program.cs (hunk 2 above)
Comment (rantifamilusi) [reply app/JobManager/Program.cs]: Yes. I was going to move that out of there.. i have done that now.

File: app/JobManager.Test/JobManager.Test.csproj (hunk 1 above)
Comment (alexjoybc) [reply app/JobManager.Test/JobManager.Test.csproj]: nothing lol, I was just having the same issue, when I create the project it target 3.0 ;-)

Comment (rantifamilusi) [reply app/JobManager.Test/JobManager.Test.csproj]: yeah ..lol

**Pull request merged**